src/transform/src/lib.rs

```diff
@@ -565,11 +567,11 @@ pub fn fold_constants_fixpoint() -> Fixpoint {
         name: "fold_constants_fixpoint",
         limit: 100,
         transforms: vec![
+            Box::new(NormalizeLets::new(false)),
```

Comment:
I don't understand why this moved. We end up with a thing that may not be normalized, but I don't see why we want that.

Reply:
Ah, sorry, I thought that the order doesn't matter from the point of view of which result we arrive at, because it's a fixpoint loop. But this may not be true if `FoldConstants` and `NormalizeLets` fight with each other for some reason. I don't see a reason why they would fight, but anyhow I'll change the order back, because we are more robust that way, because normalization is important.

(The reason why I changed the order is that in the INSERT constant scenario (and probably in other similar cases) this would settle down with one less `NormalizeLets` run with the new order. But this is not so important.)